<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineProperties</class>
 <widget class="QDialog" name="LineProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
border: 1px solid gray;
margin-top: 20px;
}

QGroupBox::title
{
    subcontrol-origin: margin;	
    border-top-left-radius: 3px;
    border-top-right-radius: 3px;
    border: 1px solid #0b477b;
    border-bottom: none;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
        stop: 0 #001520, stop: 1 #000809);
    padding: 2 2px;
    color: white;   
}</string>
     </property>
     <property name="title">
      <string>Parent Bars</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Initial Bar:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="noI"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Final Bar:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="noF"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
border: 1px solid gray;
margin-top: 20px;
}

QGroupBox::title
{
    subcontrol-origin: margin;	
    border-top-left-radius: 3px;
    border-top-right-radius: 3px;
    border: 1px solid #0b477b;
    border-bottom: none;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
        stop: 0 #001520, stop: 1 #000809);
    padding: 2 2px;
    color: white;   
}</string>
     </property>
     <property name="title">
      <string>Length</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Length:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="length"/>
        </item>
        <item>
         <widget class="QLabel" name="lengthUn">
          <property name="text">
           <string>[m]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
border: 1px solid gray;
margin-top: 20px;
}

QGroupBox::title
{
    subcontrol-origin: margin;	
    border-top-left-radius: 3px;
    border-top-right-radius: 3px;
    border: 1px solid #0b477b;
    border-bottom: none;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
        stop: 0 #001520, stop: 1 #000809);
    padding: 2 2px;
    color: white;   
}</string>
     </property>
     <property name="title">
      <string>Impedance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Zaa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zaa"/>
        </item>
        <item>
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zaai"/>
        </item>
        <item>
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn1">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Zab</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zab"/>
        </item>
        <item>
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zabi"/>
        </item>
        <item>
         <widget class="QLabel" name="label_43">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn2">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Zac</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zac"/>
        </item>
        <item>
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zaci"/>
        </item>
        <item>
         <widget class="QLabel" name="label_44">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn3">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Zan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zan"/>
        </item>
        <item>
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zani"/>
        </item>
        <item>
         <widget class="QLabel" name="label_45">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn4">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Zbb</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zbb"/>
        </item>
        <item>
         <widget class="QLabel" name="label_38">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zbbi"/>
        </item>
        <item>
         <widget class="QLabel" name="label_46">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn5">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Zbc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zbc"/>
        </item>
        <item>
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zbci"/>
        </item>
        <item>
         <widget class="QLabel" name="label_47">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn6">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Zbn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zbn"/>
        </item>
        <item>
         <widget class="QLabel" name="label_39">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zbni"/>
        </item>
        <item>
         <widget class="QLabel" name="label_48">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn7">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Zcc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zcc"/>
        </item>
        <item>
         <widget class="QLabel" name="label_40">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zcci"/>
        </item>
        <item>
         <widget class="QLabel" name="label_49">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn8">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>Zcn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zcn"/>
        </item>
        <item>
         <widget class="QLabel" name="label_41">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zcni"/>
        </item>
        <item>
         <widget class="QLabel" name="label_50">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn9">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>Znn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Znn"/>
        </item>
        <item>
         <widget class="QLabel" name="label_42">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Znni"/>
        </item>
        <item>
         <widget class="QLabel" name="label_51">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn10">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
